api/admin/controller/__init__.py

```diff
@@ -777,6 +778,21 @@ def genres(self):
 
 
 class CustomListsController(AdminCirculationManagerController):
+    class CustomListSharePostResponse(BaseModel):
+        successes: int = 0
+        failures: int = 0
+
+    class CustomListPostRequest(BaseModel):
+        name: str
+        id: int = None
```

Comment:
I think the type should be `Optional[int]`.

Reply:
Pydantic, oddly, does not seem to cover this combination of Typed variable with a None value. It just... works as None, no problem.
I'll add the `Optional` to remove ambiguity.